postman-generator/src/test/resources/apibuilder/test-service-1.json

```diff
@@ -143,6 +143,31 @@
             }
           ]
         },
+        {
+          "method": "POST",
+          "description": "Create catalog.",
+          "path": "/:organization/subcatalog",
```

Comment:
actually, this operation `POST /:organization/subcatalog`
should be moved to `subcatalog` resource  (you've created it in this PR, it already contains GET subcatalog operation)